controllers/photo.js

```diff
@@ -404,7 +404,7 @@ async function give(params) {
         // If photo has no coordinates, additionally take home position of regions
         { _id: 0, cid: 1, title_local: 1, center: 1, bbox: 1, bboxhome: 1 };
 
-    const regions = await this.call('region.getObjRegionList', { obj: photo, fields: regionFields, fromDb: !photo.geo });
+    const regions = await regionController.getObjRegionList(photo, regionFields, !photo.geo);
```

Comment:
Why do we need this changes? What are we fixing here?
`this.call()` sets the context, which might be used eventually

Reply:
We need this to make https://github.com/PastVu/pastvu/blob/master/sitemap.js#L213 line work correctly. getObjRegionList is used only in two places, in sitemap and photo controller. It does not look critical in this case to set context, but if you prefer I can rewrite its use in sitemap.